.github/workflows/test-efax-service-build-deploy.yaml

```diff
@@ -3,13 +3,19 @@ name: TEST Deploy of Efax Webservice Image
 on:
   workflow_dispatch:
     branches: [main]
```

Comment:
Is this line still needed or desired?  I don't think we ever want to (or should) build from main, only a specific tag version.

Reply:
We do not, and I removed it.